envs/example/defaults-2.0.yml

```diff
@@ -39,6 +39,7 @@ rabbitmq:
   port: 5672
   user: openstack
   cluster: false
+  nofile: 10240
```

Comment:
Ah, instead of adding it as a defaults-2.0.yml thing, we should make it a role default.